dev/spec-kitty/kittify/scripts/dashboard/static/dashboard/dashboard.css

```
@@ -0,0 +1,749 @@
+.markdown-content li {
+    margin-bottom: 0.3em;
+    line-height: 1.6;
+}
+
+.markdown-content blockquote {
+    border-left: 4px solid var(--lavender);
+    padding-left: 1em;
+    margin: 1em 0;
+    color: var(--medium-text);
+    background: #f9f9f9;
+    padding: 10px 15px;
```

Stylelint (17.4.0)

[error] Unexpected shorthand "padding" after "padding-left" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)